src/Migration.php

```diff
@@ -129,7 +129,7 @@ public function addNewMessageArea($id)
             echo "<div id='" . htmlspecialchars($this->current_message_area_id) . "'></div>";
         }
 
-        $this->displayMessage(isCommandLine() ? __('Work in progress...') : __s('Work in progress...'));
+        $this->displayMessage(__('Work in progress...'));
```

Comment:
I'm not sure this is OK

Reply:
I triple checked this one, the chain goes to displayMessage -> outputMessage -> switch outputMessageToCli or outputMessageToHtml and in the latter, htmlspecialchar is present

Reply:
:+1: